src/createStorage.js

```diff
@@ -4,37 +4,60 @@ export default function createStorage(type: 'local' | 'sync') {
   return {
     getItem(key: string) {
       return new Promise((resolve, reject) => {
-        chrome.storage[type].get(key, value => {
-          if (chrome.runtime.lastError == null) {
-            // Chrome Storage returns the value in an Object of with its original key. Unwrap the
-            // value from the returned Object to match the `getItem` API.
-            resolve(value[key]);
-          } else {
-            reject();
-          }
-        });
+        if (typeof chrome !== 'undefined') {
```

Comment:
Since `browser.storage` is the standard going forward, let's prefer it over the `chrome` global. Check for the existence of `browser` and fallback to `chrome` otherwise.